### ISSUE TITLE
Can a non-ROS system robotic arm use this method? Are there any configuration requirements?

### ISSUE THREAD

**Issue by a-liukun** [graspnet/anygrasp_sdk#6]
The Unitree Z1 robotic arm does not support ROS and uses UDP communication. Can AnyGrasp be used? Also, can this algorithm be used on an integrated graphics card?

**Comment by chenxi-wang** [comment]
Yes, this program is implemented using Python and PyTorch. So you can launch it without ROS.
